rigex-front-end/src/pages/Landing/Landing.js

```diff
@@ -1,13 +1,21 @@
 import React from 'react';
-import { Route } from 'react-router-dom';
+import { Route, useLocation, Redirect } from 'react-router-dom';
 
 import routePaths from '../../constants/routePaths';
 import { Rig } from '../index';
 
 const Landing = () => {
+  const { pathname } = useLocation();
+  const regex = /^[a-z0-9]+$/i;
+
+  let isValidRigPath =
+    pathname.includes(routePaths.landing + '/rig') &&
+    regex.test(pathname.slice(13));
+
   return (
     <div>
       <h1>Landing Works!</h1>
+      {!isValidRigPath && <Redirect to={routePaths.landing} />}
```

Comment:
How can you solve this using a function that can live in the utils folder while keeping in mind that there may be other nested paths off landing in the future